### PR TITLE
Doc addition

### PR DESCRIPTION
**Pull request by RashelleKosib**
info for all index file added

### CODE REVIEW

File: docs/contribution-guide.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ it's faster, and cheaper on your runtime
 
 All component APIs must be strongly typed, with `any` not accepted. it might be harder to write, but its much easier to use.
 
+### Component Drivers
+**component.driver.ts**
+
+Note that in the root there is a drivers folder containing index files according to your testing platform. It is an index file each component's driver export.
+If you are trying to import a components driver outside of wix-ui-core, please use it.
+
```
[the remark this answers is not in the document]
Comment (JoMarton) [reply docs/contribution-guide.md]: Agree with all @lbelinsk comments. A code example is worth `inf` amount of words.

File: docs/contribution-guide.md [hunk 2]
```diff
@@ -79,8 +85,14 @@ a component importing the following stylable file:
 
 ```
 
+Note that in the root there is an index.st.css file. It is an index file for all component's style exports.
```
Comment (JoMarton) [inline docs/contribution-guide.md]: I would add to the code snippet

File: docs/usage-guide.md [hunk 3]
```diff
@@ -67,7 +67,8 @@ function App() {
 }
 
 ```
-
+Note that in the root there is an index.st.css file. It is an index file for all component's style exports.
```
Comment (JoMarton) [inline docs/usage-guide.md]: Is this relevant here?

@lbelinsk does this example conform to the current theming structure

File: docs/contribution-guide.md [hunk 4]
```diff
@@ -79,8 +85,14 @@ a component importing the following stylable file:
...
+If you are trying to import a style outside of wix-ui-core, please use it.
+
 is exposing both the `btn1` class and `btn2` class to stylable so they can be customized.
 
+Note that in the root there is an index.st.css file. It is an index file fore all component's style exports.
```
[the remark this answers is not in the document]
Comment (RashelleKosib) [reply docs/contribution-guide.md]: not duplication :) i was talking here about the index.st.css file,

**Comment by RashelleKosib** [comment]
@lbelinsk @RashelleKosib  PR fixed were pushed, tnx for your notes.

**Comment by lbelinsk** [comment]
This is not the way we decided to go in the end with the themed libraries since the component's will be taken from `wix-ui-backoffice` (or wix-style-react) and not from the `wix-ui-core`.
@JoMarton correct me if I am wrong

**Comment by JoMarton** [comment]
@lbelinsk 
I think that if that is the case the PR is still ok no?

**Comment by RashelleKosib** [comment]
@shlomitc  please merge

**Pull request merged**